lib/Models/Catalog/CatalogItems/GeoJsonCatalogItem.ts

```diff
@@ -145,7 +145,7 @@ class GeoJsonCatalogItem
       // FeatureCollection to JsonValue
       jsonData = mergeFeatureCollections(
         featureCollections
-      ) as any as JsonValue;
+      ) as unknown as JsonValue;
```

Comment:
Since this change removes most `as any as`, I changed the 3 remaining ones into `as unknown as` to fix some complaints from ESLint about using the any type.